### PR TITLE
Add usage, setup and transfer information to the documentation

### PR DESCRIPTION
**Pull request by gridhead**
Add usage, setup and transfer information to the documentation

### CODE REVIEW

File: data/05.png
Comment (gridhead) [inline data/05.png:1]: This is not the correct image.

This needs to be changed.

File: README.md [hunk 1]
```diff
@@ -1,2 +1,124 @@
 # protop2g
 A prototype project assets importer that moves repositories from Pagure to GitLab
+
+## Usage
+
+### Installation
+
+1. Ensure that you have `git`, `python3`, `virtualenv` and `poetry` installed.
+    ```
```
Comment (samyak-jn) [inline README.md:9]:
```suggestion
    ```

```
Add a line after every point and before the code/command snippet so there will be a line break between the two. (For clean cosmetic purposes)

File: README.md [hunk 2]
```diff
@@ -1,2 +1,124 @@
...
+    $ sudo dnf install git python3 virtualenv poetry --setopt=install_weak_deps=False
+    ```
+2. Clone the repository to the local storage and make it the present working directory.
+    ```
```
Comment (samyak-jn) [inline README.md:13]:
```suggestion
    ```

```

Similar here^

File: README.md [hunk 3]
```diff
@@ -1,2 +1,124 @@
...
+    Cloning into 'protop2g'...
+    remote: Enumerating objects: 118, done.
+    remote: Counting objects: 100% (118/118), done.
+    remote: Compressing objects: 100% (78/78), done.
+    remote: Total 118 (delta 48), reused 94 (delta 31), pack-reused 0
+    Receiving objects: 100% (118/118), 56.38 KiB | 4.34 MiB/s, done.
+    Resolving deltas: 100% (48/48), done.
+    ```
+3. Create and activate a Python virtual environment in that directory.
+    ```
```
Comment (samyak-jn) [inline README.md:30]:
```suggestion
    ```

```

Similar here^

File: README.md [hunk 4]
```diff
@@ -1,2 +1,124 @@
...
+    Sample output  
+    ```
+    created virtual environment CPython3.11.5.final.0-64 in 143ms
+      creator CPython3Posix(dest=/home/archdesk/Projects/protop2g/venv, clear=False, no_vcs_ignore=False, global=False)
+      seeder FromAppData(download=False, pip=bundle, setuptools=bundle, wheel=bundle, via=copy, app_data_dir=/home/archdesk/.local/share/virtualenv)
+        added seed packages: pip==23.2.1, setuptools==68.0.0, wheel==0.41.1
+      activators BashActivator,CShellActivator,FishActivator,NushellActivator,PowerShellActivator,PythonActivator
+    ```
+4. Check the project configuration's validity and then install the project dependencies.
+    ```
```
Comment (samyak-jn) [inline README.md:45]:
```suggestion
    ```

```

Similar here^

File: README.md [hunk 5]
```diff
@@ -1,2 +1,124 @@
...
+      • Installing requests (2.28.2)
+      • Installing tqdm (4.64.1)
+    
+    Installing the current project: protop2g (0.1.0)
+    ```
+
+## Setup
+
+1. Using a browser of your choice, open up [Pagure](https://pagure.io) and login to your account.
+    ![](data/01.png)
```
Comment (samyak-jn) [inline README.md:86]: Similar here, the picture and the point doesn't seem to have a line break^

File: README.md [hunk 6]
```diff
@@ -1,2 +1,124 @@
...
+2. Click on your profile display picture and then, head over to the [Account Settings](https://pagure.io/settings) page.
+    ![](data/02.png)
```
Comment (samyak-jn) [inline README.md:88]: Similar here, the picture and the point doesn't seem to have a line break^

File: README.md [hunk 7]
```diff
@@ -1,2 +1,124 @@
...
+3. Under the [API Keys](https://pagure.io/settings#nav-api-tab) section, click on the [Create new API key](https://pagure.io/settings/token/new) button in the top right corner.
+    ![](data/03.png)
```
Comment (samyak-jn) [inline README.md:90]: Similar here, the picture and the point doesn't seem to have a line break^

File: README.md [hunk 8]
```diff
@@ -1,2 +1,124 @@
...
+4. As this is the source namespace, check all the ACLs that are required to read the asset information associated with a repository, set a safe expiration date for the API token and write an appropriate description for its usage before clicking on the [Create](#) button.
+    ![](data/04.png)
```
Comment (samyak-jn) [inline README.md:92]: Similar here, the picture and the point doesn't seem to have a line break^

File: README.md [hunk 9]
```diff
@@ -1,2 +1,124 @@
...
+5. Make note of the API token generated and ensure that they are not shared with others or used for a different purpose.
+    ![](data/05.png)
```
Comment (samyak-jn) [inline README.md:94]: Similar here, the picture and the point doesn't seem to have a line break^

File: README.md [hunk 10]
```diff
@@ -1,2 +1,124 @@
...
+6. Head over to a repository that the currently logged-in username has at least a READ access to.
+    ![](data/06.png)
```
Comment (samyak-jn) [inline README.md:96]: Similar here, the picture and the point doesn't seem to have a line break^

File: README.md [hunk 11]
```diff
@@ -1,2 +1,124 @@
...
+7. Make note of the source namespace in the format of `HOLDER/REPONAME` where the `HOLDER` can be a group or a sole user.  
+    For example, in case of a repository located at `https://pagure.io/fedora-infra/ansible` - the source namespace would be `fedora-infra/ansible`.
+    ![](data/07.png)
```
Comment (samyak-jn) [inline README.md:99]: Similar here, the picture and the point doesn't seem to have a line break^

File: README.md [hunk 12]
```diff
@@ -1,2 +1,124 @@
...
+8. In another browser tab or window, open up [GitLab](https://gitlab.com/) and login to your account.
+    ![](data/08.png)
```
Comment (samyak-jn) [inline README.md:101]: Similar here, the picture and the point doesn't seem to have a line break^

File: README.md [hunk 13]
```diff
@@ -1,2 +1,124 @@
...
+9. Click on the [New project/repository](https://gitlab.com/projects/new) option from the sidebar of the profile page and then, click on the [Create blank project](https://gitlab.com/projects/new#blank_project) option.
+    ![](data/09.png)
```
Comment (samyak-jn) [inline README.md:103]: Similar here, the picture and the point doesn't seem to have a line break^

File: README.md [hunk 14]
```diff
@@ -1,2 +1,124 @@
...
+10. Create a new empty repository which will act as the destination for the asset transfer. It is recommended to have the same name as the source namespace to avoid confusion, but it is not strictly required.
+    ![](data/10.png)
```
Comment (samyak-jn) [inline README.md:105]: Similar here, the picture and the point doesn't seem to have a line break^

File: README.md [hunk 15]
```diff
@@ -1,2 +1,124 @@
...
+11. Head over to the created repository and make note of the `Project ID`.  
+    For example in this case, it is `42823949` for the destination repository named `gridhead/protop2g-test`.  
+    ![](data/11.png)
```
Comment (samyak-jn) [inline README.md:108]: Similar here, the picture and the point doesn't seem to have a line break^

File: README.md [hunk 16]
```diff
@@ -1,2 +1,124 @@
...
+12. Expand the sidebar to head over to the [Access Tokens](https://gitlab.com/gridhead/protop2g-test/-/settings/access_tokens) section from the `Settings` section.
+    ![](data/12.png)
```
Comment (samyak-jn) [inline README.md:110]: Similar here, the picture and the point doesn't seem to have a line break^

File: README.md [hunk 17]
```diff
@@ -1,2 +1,124 @@
...
+13. In the [Project Access Token](https://gitlab.com/gridhead/protop2g-test/-/settings/access_tokens) page, click on the [Add new token](#) button to begin creating a new access token.
+    ![](data/13.png)
```
Comment (samyak-jn) [inline README.md:112]: Similar here, the picture and the point doesn't seem to have a line break^

File: README.md [hunk 18]
```diff
@@ -1,2 +1,124 @@
...
+14. As this is the destination namespace, check all the scopes that are required to write the asset information associated with a repository, pick an appropriate role, set a safe expiration date and write an appropriate description for its usage before clicking on the [Create project access token](#) button.
+    ![](data/14.png)
```
Comment (samyak-jn) [inline README.md:114]: Similar here, the picture and the point doesn't seem to have a line break^

File: README.md [hunk 19]
```diff
@@ -1,2 +1,124 @@
...
+15. Make note of the API token generated and ensure that they are not shared with others or used for a different purpose.
+    ![](data/15.png)
```
Comment (samyak-jn) [inline README.md:116]: Similar here, the picture and the point doesn't seem to have a line break^

**Review by samyak-jn — APPROVED** [review]
Hi @gridhead!
Thanks for this, On looking into the documentation, everything is well written!
I have just some cosmetic changes for you to incorporate for clean purposes! Approving from my end :rocket:

**Comment by gridhead** [comment]
Thanks for the review, @samyak-jn !

**Pull request merged**